### ISSUE TITLE
filestore add bug when running daemon offline

### ISSUE THREAD

**Issue by jefft0** [ipfs-filestore/go-ipfs#15]
go-ipfs version: 0.4.4-dev-b381491
Repo version: 4
System version: amd64/windows
Golang version: go1.5.3

Type: bug.
Area: commands

Description: If I start the daemon with `ipfs daemon --offline` and run `ipfs filestore add myfile` I get the following error:

```
[0;37m18:39:34.934 ←[31mERROR ←[0;34mcommands/h: ←[0mno file information for block ←[0;37mclient.go:247←[0m
```

**Comment by kevina** [comment]
Confirmed.  Will look into it.

**Comment by kevina** [comment]
This should be fixed now (in 69286925a5b6a5e8a04f9b36cbdded904a7c9248).  Let me know if it's not.
